file_picker/__init__.py

```diff
@@ -1,29 +1,13 @@
+# Aliases allow users to import directly from file_picker.*
+try:
+    from file_picker.views import FilePickerBase, ImagePickerBase  # noqa
+    from file_picker.sites import site, FilePickerSite  # noqa
+    from file_picker import widgets  # noqa
+except ImportError:
+    # setup.py can't access __version__ unless django and deps are installed
+    # We'll deprecate these aliases now, and they should be removed in the next version.
```

Comment:
I wonder if there's any way to catch usage of these aliases and issue deprecation warnings?

Reply:
I've thought and experimented for a while and don't see a good way to do this. The `__init__.py` file will always be run before any other files within that module, so the warning would fire whether you did `from file_picker import site` or `from file_picker.sites import site`.

The main reason that I wanted to deprecate the aliases was so that I could reuse the version number here and in `setup.py`. At this point, I don't think it is worth it, and I think we should leave the aliases alone, and just hardcode the version in setup.py.

What do you think, @dpoirier?